cmake/TestImpactFramework/LYTestImpactFramework.cmake

```diff
@@ -439,6 +441,45 @@ function(ly_test_impact_write_config_file CONFIG_TEMPLATE_FILE BIN_DIR)
     message(DEBUG "Test impact framework post steps complete")
 endfunction()
 
+function(ly_test_impact_write_pytest_file)
+    set(config_array "")
+    foreach(config_type ${LY_CONFIGURATION_TYPES})
+        if(NOT "${config_type}" STREQUAL "release")
+            list(APPEND config_array "\"${config_type}\":\"${LY_TEST_IMPACT_WORKING_DIR}/${config_type}/Persistent/tiaf.json\"")
```

Comment:
`Persistent` should be grabbed via a cmake var (will require some rejigging for how vars like `LY_TEST_IMPACT_PERSISTENT_DIR` are compiled).

Reply:
Sure, lets discuss on monday as I'm not sure the best way to do this, generator expressions introduce some weird behaviour and I don't want to needlessly duplicate variable declarations.